<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_MsgBox</class>
 <widget class="QWidget" name="Form_MsgBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>117</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_tips">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tips_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_ok</sender>
   <signal>clicked()</signal>
   <receiver>Form_MsgBox</receiver>
   <slot>btnOKSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btnOKSlot()</slot>
 </slots>
</ui>
